tsconfig.json

```diff
@@ -0,0 +1,28 @@
+{
+    "compilerOptions": {
+        "target": "es2015",
+        "module": "commonjs",
+        "moduleResolution": "node",
+        "lib": [
+            "es2017",
+            "dom"
+        ],
+        "types": [
```

Comment:
does it need [@types/puppeteer](https://www.npmjs.com/package/@types/puppeteer) here?

Reply:
I tested that one. It seems it doesn't need `@types/puppeteer` in the end. I think the reason is of because `@types/cheerio` does not export some any declared types, but `CheerioAPI`:

```typescript
interface CheerioStatic extends CheerioSelector { }
```
While `@types/puppeteer` does export everything:
```typescript
export interface Browser extends EventEmitter, TargetAwaiter { }
```
Correct me if I'm wrong.

Reply:
indeed, cheerio uses global declarations